validator/validate.js

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/env node
+
+var fs = require("fs");
```

Comment:
`var` missing in the rest of declarations

Reply:
fixed